lisa/operating_system.py

```diff
@@ -1034,6 +1034,21 @@ def _package_exists(self, package: str) -> bool:
 
         return False
 
+    def install_epel(self) -> None:
+        # Extra Packages for Enterprise Linux (EPEL) is a special interest group
+        # (SIG) from the Fedora Project that provides a set of additional packages
+        # for RHEL (and CentOS, and others) from the Fedora sources.
+
+        major = self._node.os.information.version.major
+        assert_that(major).described_as(
+            "Fedora/RedHat version must be greater than 7"
+        ).is_greater_than_or_equal_to(7)
+        epel_release_rpm_name = f"epel-release-latest-{major}.noarch.rpm"
+        self._install_packages(
```

Comment:
Avoid calling private method if there is no special reason.

Reply:
on update_packages there was a reason, to avoid passing in any package names. I just wanted a generic yum update and update_packages requires a no-op string list or empty string. _update_packages allows None so I went with that one. I'll switch install_packages though there's no reason for that one.